docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-linux.md

```diff
@@ -10,9 +10,101 @@ If you do not have a DirectX 11 capable GPU in your system (for example if you a
 # <a name="Headless_Servers">Headless Servers</a>
 
 ## Using Docker
-[pg9182/northstar_dedicated](https://github.com/pg9182/northstar-dedicated) : Docker image with server management tools and better performance 
 
-Check the `README` page for instructions on the appropriate repository to set up your server.
+The preferred method for running a Northstar server in Docker is using pg9182's image which can be found in this GitHub repo: [https://github.com/pg9182/northstar-dedicated](https://github.com/pg9182/northstar-dedicated)
+
+### Prerequisites
+
+| Requirement | Description                                             |
+| ----------- | ------------------------------------------------------- |
+| Kernel      | Linux 5.3+, but 4.9+ should work                        |
+| CPU         | x86\_64, at least 3 cores minimum                       |
+| RAM         | 2GB (Typically peaks at 1.6GB)                          |
+| Disk        | 5GB                                                     |
+| Network     | Recommended at least 7-20Mbps up                        |
+| Docker      | Have Docker and Docker-Compose installed on the machine |
+
+### Installation
+
+#### Install Docker and Docker-Compose
+
+If you don't already have Docker and Docker-Compose installed, you can learn how to install them [here](https://docs.docker.com/engine/install/) for your distro. Once you have them installed, you can move on to the next section.
+
+### Prep Titanfall Server files
+
+You will need obtain Titanfall2's game files, typically by copying over your Titanfall2 installation folder and placing it on the Linux machine. Most of these files are for SP and therefore we can delete them from the installation folder and prune the file size down to \~5GB.
+
+Easiest way to do this is to copy the entire Titanfall2 folder to the server and delete the [following files](https://github.com/pg9182/northstar-dedicated#reducing-the-size-), or alternatively download and run [this shell script](https://gist.github.com/pg9182/9a962adbfc27e93237cd14e4523c9da8) to download only the files necessary to run a server.
+
+#### Copy Titanfall Folder over to the Linux Machine
+
+1. Access the files on your Linux machine using tools like [WinSCP](https://winscp.net/eng/download.php) or any tool that allows for SSH file transfer, or transfer files via USB drive if you can physically access your server.
+2. Navigate to the folder where you want to store the files. You can put them into `~/Titanfall2` for example.
+3. Copy the newly pruned Titanfall folder to the server.
+
+![Game files copied to `/mnt/Titanfall` as an example](https://i.postimg.cc/15HbbzFr/image.png)
+
+#### Copy mods to server
+
+If you have configured some mods,these can be placed at a similar location, like `~/mods` for example
+![Mods placed inside /mnt/mods](https://i.postimg.cc/tRD5jnrJ/image.png)
```

Comment:
again, `/mnt/` is a bad place and images should be hosted first party not external

Reply:
(this is how it was when I got to iiiiit)